test/command/suite/sharding/logical_count/log/select.expected

```diff
@@ -4,10 +4,14 @@ table_create Logs_20150709 TABLE_NO_KEY
 [[0,0.0,0.0],true]
 column_create Logs_20150709 timestamp COLUMN_SCALAR Time
 [[0,0.0,0.0],true]
+load --table Logs_20150709
+[
+{"timestamp": "2015-07-09 20:00:00"}
+]
+[[0,0.0,0.0],1]
 log_level --level debug
 [[0,0.0,0.0],true]
 logical_count Logs timestamp   --min "2015/07/09 00:00:00"   --max "2015/07/09 12:00:00"
 [[0,0.0,0.0],0]
-#|d| [logical_count][select] <Logs_20150709>: no range index
```

Comment:
Currently the debug log `no range index` is not outputted.
I will fix them in another PR because we need to modify StreamShardExecutor (https://github.com/groonga/groonga/blob/master/plugins/sharding/stream_shard_executor.rb#L74) for fixing it and it affects to not only `logical_count` but also `logical_range_filter`.

Reply:
This is resolved by #1349